CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ endif()
 #  Build defaults for projects by the Realistic Graphics Lab
 # ----------------------------------------------------------
 
-include(ext/cmake-defaults/CMakeLists.txt)
```

Comment:
I am curious to understand this change: are there situations in which this `include` directive would fail? My understanding was that such things are always relative to` $CMAKE_CURRENT_SOURCE_DIR`.

Reply:
You're right, apologies! Good catch :)